Makefile

```diff
@@ -140,6 +148,20 @@ tools: host-tools check-env $(BUILD)/$(TARGET)/CMakeCache.txt
 toit-tools: tools download-packages
 	(cd $(BUILD)/$(TARGET) && ninja build_toit_tools)
 
+# Rebuilds the SDK using only Ninja, without rebuilding the
+# Ninja files with Cmake.
+.PHONY: fast
+fast: fast-tools fast-toit-tools
```

Comment:
typically we have `-no-X` versions of the clauses. For example `esp32-no-env` for the `esp32` rule.
I think `tools-no-sync` would fit in that category. Also makes the command-line completion easier.

Reply:
Obviously should be `tools-no-cmake`.